kibble/api/languages.go

```diff
@@ -87,3 +94,25 @@ func formatPathLocale(code string) string {
 	dashedCode := strings.ReplaceAll(code, "_", "-")
 	return strings.ToLower(dashedCode)
 }
+
+func loadFromConfig(cfg *models.Config, site *models.Site) {
```

Comment:
Should probably call this `loadLanguagesFromConfig` as its a package-wide function that is a tad too vague about its intentions.

Reply:
Ah, Go has package-wide visibility of funcs... 👍